vignettes/blueprint/config.yml

```diff
@@ -0,0 +1,52 @@
+default:
+  defaultOrg: insightsengineering
+  # All the repos are by assumed to live inside the org `defaultOrg` except for these exceptions
+  githubOrgExceptions:
+    - mmrm: openpharma
+  packages:
+    Teal:
+      color: "#258fdb"
+      packages:
+        - teal.logger
+        - teal.code
+        - teal.data
+        - teal.slice
+        - teal.transform
+        - teal.widgets
+        - teal.reporter
+        - teal
+    Modules:
+      color: "#f24b4b"
+      packages:
+        - teal.osprey
+        - teal.goshawk
+        - teal.modules.clinical
+        - teal.modules.helios
```

Comment:
Question: Do we need `teal.modules.helios` here or do we add it here after the public release is made? If we only use public repos we don't need GitHub token to access the `staged_dependencies.yaml` and the pkgdown site will build without any issues.